### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by MustaphaLimam**
Tutoriel partie 6

### CODE REVIEW

File: docs/tutorial/part-six/index.md [hunk 1]
```diff
@@ -1,30 +1,29 @@
 ---
-title: Transformer plugins
+title: Plugins transformateur
```
Comment (dantereve) [inline docs/tutorial/part-six/index.md]:
```suggestion
title: Plugins transformateurs
```

File: docs/tutorial/part-six/index.md [hunk 2]
```diff
@@ -1,30 +1,29 @@
...
 typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-> This tutorial is part of a series about Gatsby’s data layer. Make sure you’ve gone through [part 4](/tutorial/part-four/) and [part 5](/tutorial/part-five/) before continuing here.
+> Ce tutoriel fait partie d’une série sur la couche de données de Gatsby. Assurez-vous d'avoir traversé la [partie 4](/tutorial/part-four/) et la [partie 5](/tutorial/part-five/) avant de continuer ici.
 
-## What's in this tutorial?
+## Qu'est-ce qu'il y a dans ce tutoriel?
```
Comment (dantereve) [inline docs/tutorial/part-six/index.md]:
```suggestion
## Que contient ce tutoriel ?
```

File: docs/tutorial/part-six/index.md [hunk 3]
```diff
@@ -1,30 +1,29 @@
...
 
-The previous tutorial showed how source plugins bring data _into_ Gatsby’s data system. In this tutorial, you'll learn how transformer plugins _transform_ the raw content brought by source plugins. The combination of source plugins and transformer plugins can handle all data sourcing and data transformation you might need when building a Gatsby site.
+Le tutoriel précédent montrait comment les plugins source apportent les données _dans_ le système de données de Gatsby. Dans ce tutoriel, vous apprendrez comment les plugins transformateur _transforment_ le contenu brut apporté par les plugins sources. La combinaison de plugins source et de plugins transformateur peut prendre en charge toutes les sources de données et la transformation de données dont vous pourriez avoir besoin lors de la construction d'un site Gatsby.
```
Comment (dantereve) [inline docs/tutorial/part-six/index.md]:
```suggestion
Le tutoriel précédent montrait comment les plugins source apportent les données _dans_ le système de données de Gatsby. Dans ce tutoriel, vous apprendrez comment les plugins transformateurs _transforment_ le contenu brut apporté par les plugins sources. La combinaison de plugins sources et de plugins transformateurs peut prendre en charge toutes les sources de données et la transformation de données dont vous pourriez avoir besoin lors de la construction d'un site Gatsby.
```

File: docs/tutorial/part-six/index.md [hunk 4]
```diff
@@ -1,30 +1,29 @@
...
 
-## Transformer plugins
+## Plugins transformateur
```
Comment (dantereve) [inline docs/tutorial/part-six/index.md]:
```suggestion
## Plugins transformateurs
```

File: docs/tutorial/part-six/index.md [hunk 5]
```diff
@@ -1,30 +1,29 @@
...
 
-Often, the format of the data you get from source plugins isn't what you want to
-use to build your website. The filesystem source plugin lets you query data
-_about_ files but what if you want to query data _inside_ files?
+Souvent, le format des données que vous obtenez des plugins source n'est pas ce que vous voulez.
```
Comment (dantereve) [inline docs/tutorial/part-six/index.md]:
```suggestion
Souvent, le format des données que vous obtenez des plugins sources n'est pas ce que vous voulez.
```

File: docs/tutorial/part-six/index.md [hunk 6]
```diff
@@ -1,30 +1,29 @@
...
+pour construire votre site Web. Le plugin source du système de fichiers vous permet d'interroger les données
+_à propos_ des fichiers mais que faire si vous voulez interroger des données _dans_ des fichiers?
```
Comment (dantereve) [inline docs/tutorial/part-six/index.md]:
```suggestion
_à propos_ des fichiers, mais que faire si vous voulez interroger des données _dans_ des fichiers?
```

File: docs/tutorial/part-six/index.md [hunk 7]
```diff
@@ -1,30 +1,29 @@
...
 
-To make this possible, Gatsby supports transformer plugins which take raw
-content from source plugins and _transform_ it into something more usable.
+Pour rendre cela possible, Gatsby supporte les plugins transformateur qui prennent le contenu brut des plugins
```
Comment (dantereve) [inline docs/tutorial/part-six/index.md]:
```suggestion
Pour rendre cela possible, Gatsby supporte les plugins transformateurs qui prennent le contenu brut des plugins 
```

File: docs/tutorial/part-six/index.md [hunk 8]
```diff
@@ -1,30 +1,29 @@
...
+_à propos_ des fichiers mais que faire si vous voulez interroger des données _dans_ des fichiers?
 
-To make this possible, Gatsby supports transformer plugins which take raw
-content from source plugins and _transform_ it into something more usable.
+Pour rendre cela possible, Gatsby supporte les plugins transformateur qui prennent le contenu brut des plugins 
+source et les _transforment_ en quelque chose de plus utilisable.
 
-For example, markdown files. Markdown is nice to write in but when you build a
-page with it, you need the markdown to be HTML.
+Par exemple, les fichiers markdown. Markdown est agréable à écrire, mais lorsque vous construisez une page avec lui, vous avez besoin du markdown pour être HTML.
```
Comment (dantereve) [inline docs/tutorial/part-six/index.md]:
```suggestion
Par exemple, les fichiers markdown. Markdown est agréable à écrire, mais lorsque vous construisez une page avec lui, vous avez besoin que le markdowm soit du HTML.
```

File: docs/tutorial/part-six/index.md [hunk 9]
```diff
@@ -1,30 +1,29 @@
...
 
-Add a markdown file to your site at
-`src/pages/sweet-pandas-eating-sweets.md` (This will become your first markdown
-blog post) and learn how to _transform_ it to HTML using transformer plugins and
+Ajoutez un fichier markdown à votre site à l'adresse suivante
```
Comment (dantereve) [inline docs/tutorial/part-six/index.md]:
```suggestion
Ajoutez un fichier markdown à votre site à l'emplacement suivant :
```

File: docs/tutorial/part-six/index.md [hunk 10]
```diff
@@ -1,30 +1,29 @@
...
+`src/pages/sweet-pandas-eating-sweets.md` (Ceci deviendra votre premier markdown
```
Comment (dantereve) [inline docs/tutorial/part-six/index.md]:
```suggestion
`src/pages/sweet-pandas-eating-sweets.md` (Ceci deviendra votre premier post de blog
```

File: docs/tutorial/part-six/index.md [hunk 11]
```diff
@@ -1,30 +1,29 @@
...
+blog post) et apprenez comment _transformer_ cela en HTML à l'aide de plugins transformateur et de
```
Comment (dantereve) [inline docs/tutorial/part-six/index.md]:
```suggestion
en markdown) et apprenez comment _transformer_ cela en HTML à l'aide de plugins transformateur et de
```

File: docs/tutorial/part-six/index.md [hunk 12]
```diff
@@ -40,19 +39,18 @@ Here's a video of a panda eating sweets.
...
 ```
 
-Once you save the file, look at `/my-files/` again—the new markdown file is in
-the table. This is a very powerful feature of Gatsby. Like the earlier
-`siteMetadata` example, source plugins can live-reload data.
-`gatsby-source-filesystem` is always scanning for new files to be added and when
-they are, re-runs your queries.
+Une fois que vous avez sauvegardé le fichier, regardez encore une fois `/my-files/`—le nouveau fichier markdown est dans
+la table. C'est une caractéristique très puissante de Gatsby. Comme le premier exemple de
+`siteMetadata`, les plugins source peuvent recharger les données en direct.
```
Comment (dantereve) [inline docs/tutorial/part-six/index.md]:
```suggestion
`siteMetadata`, les plugins sources peuvent recharger les données en direct.
```

File: docs/tutorial/part-six/index.md [hunk 13]
```diff
@@ -79,31 +77,29 @@ module.exports = {
 }
 ```
 
-Restart the development server then refresh (or open again) GraphiQL and look
-at the autocomplete:
+Redémarrez le serveur de développement puis rafraîchissez (ou ou ouvrez à nouveau) GraphiQL et regardez
```
Comment (dantereve) [inline docs/tutorial/part-six/index.md]:
```suggestion
Redémarrez le serveur de développement puis rafraîchissez (ou ouvrez à nouveau) GraphiQL et regardez 
```

File: docs/tutorial/part-six/index.md [hunk 14]
```diff
@@ -79,31 +77,29 @@ module.exports = {
...
+à l'auto-complétion:
 
 ![markdown-autocomplete](markdown-autocomplete.png)
 
-Select `allMarkdownRemark` again and run it as you did for `allFile`. You'll
-see there the markdown file you recently added. Explore the fields that are
-available on the `MarkdownRemark` node.
+Sélectionnez à nouveau `allMarkdownRemark` et exécutez-le comme vous l'avez fait pour `allFile`. Vous allez
+y voir le fichier de démarque que vous avez récemment ajouté. Explorez les domaines qui sont
+disponible sur le nœud `MarkdownRemark`.
```
Comment (dantereve) [inline docs/tutorial/part-six/index.md]:
```suggestion
disponibles sur le nœud `MarkdownRemark`.
```

File: docs/tutorial/part-six/index.md [hunk 15]
```diff
@@ -79,31 +77,29 @@ module.exports = {
...
 
 ![markdown-query](markdown-query.png)
 
-Ok! Hopefully, some basics are starting to fall into place. Source plugins bring
-data _into_ Gatsby's data system and _transformer_ plugins transform raw content
-brought by source plugins. This pattern can handle all data sourcing and
-data transformation you might need when building a Gatsby site.
+Ok! Avec un peu de chance, les bases commencent à se mettre en place. Les plugins source amènent les données
```
Comment (dantereve) [inline docs/tutorial/part-six/index.md]:
```suggestion
Ok ! Avec un peu de chance, les bases commencent à se mettre en place. Les plugins sources amènent les données
```

File: docs/tutorial/part-six/index.md [hunk 16]
```diff
@@ -79,31 +77,29 @@ module.exports = {
...
+_dans_ le système de données de Gatsby et les plugins _transformateur_ transforment le contenu brut apporté par les plugins source. Ce modèle peut gérer toutes les sources de données et la transformation de données dont vous pourriez avoir besoin lors de la construction d'un site Gatsby.
```
Comment (dantereve) [inline docs/tutorial/part-six/index.md]:
```suggestion
_dans_ le système de données de Gatsby et les plugins _transformateurs_ transforment le contenu brut apporté par les plugins source. Ce modèle peut gérer toutes les sources de données et la transformation de données dont vous pourriez avoir besoin lors de la construction d'un site Gatsby.
```

File: docs/tutorial/part-six/index.md [hunk 17]
```diff
@@ -79,31 +77,29 @@ module.exports = {
...
-## Create a list of your site's markdown files in `src/pages/index.js`
+## Créez une liste des fichiers markdown de votre site dans `src/pages/index.js`
 
-Now you'll have to create a list of your markdown files on the front page. Like many
-blogs, you want to end up with a list of links on the front page pointing to each
-blog post. With GraphQL you can _query_ for the current list of markdown blog
-posts so you won't need to maintain the list manually.
+Vous devez maintenant créer une liste de vos fichiers markdown sur la page d'accueil. Comme beaucoup de blogs, vous 
+voulez finir avec une liste de liens sur la page d'accueil pointant vers chaque article du blog. Avec GraphQL 
+vous pouvez _interroger_ pour la liste courante des markdown des articles du blog afin de ne pas avoir besoin de
```
Comment (dantereve) [inline docs/tutorial/part-six/index.md]:
```suggestion
vous pouvez _interroger_ la liste courante des articles markdown du blog afin de ne pas avoir besoin de 
```

File: docs/tutorial/part-six/index.md [hunk 18]
```diff
@@ -169,11 +165,11 @@ export const query = graphql`
 `
 ```
 
-Now the frontpage should look like:
+Maintenant, la page d'accueil devrait ressembler:
```
Comment (dantereve) [inline docs/tutorial/part-six/index.md]:
```suggestion
Maintenant, la page d'accueil devrait ressembler à :
```

File: docs/tutorial/part-six/index.md [hunk 19]
```diff
@@ -169,11 +165,11 @@ export const query = graphql`
...
 
 ![frontpage](frontpage.png)
 
-But your one blog post looks a bit lonely. So let's add another one at
+Mais votre seul article du blog semble un peu isolé. Ajoutons-en un autre à l'adresse
```
Comment (dantereve) [inline docs/tutorial/part-six/index.md]:
```suggestion
Mais votre seul article du blog semble un peu isolé. Ajoutons-en un autre à l'emplacement
```

**Review by dantereve — CHANGES_REQUESTED** [review]
Awesome work @MustaphaLimam! Many thanks! I've just requested some minor changes (mainly typos). You can apply them directly from this PR if it's fine for you. Don't hesitate to comment on those requests if you disagree.   
Once this is done I will merge this great PR 💪.

**Comment by MustaphaLimam** [comment]
Glad to help out ! Thanks for your feedback ! Can't wait to contribute more !

**Review by dantereve — APPROVED** [review]

**Comment by dantereve** [comment]
It's all good, your PR is now merged!
Let's celebrate your first contribution to Gatsby! 🥳

**Pull request merged**